src/utils/helpers.js

```diff
@@ -201,6 +201,20 @@ const helpers = {
    */
   isMainnet() {
     return hathorLib.network.getNetwork().name === 'mainnet';
+  },
+
+  /**
+   * Get file extension
+   *
+   * @param {string} file File name
+   *
+   * @return {string} extension of the file
+   * @memberof Helpers
+   * @inner
+   */
+  getFileExtension(file) {
+    const parts = file.split('.');
+    return parts[parts.length - 1];
```

Comment:
If the file has no extension at all (e.g. `myfile`), it will return the base name instead of the extension. Maybe add the following?
```nodejs
if (parts.length == 1) {
  return '';
}
```

Reply:
Sure! I think we can still improve given that the file is an URI. But don't think we have to care about now.

Reply:
If things get more complicated, maybe we should use a library that parses URI. But it doesn't seem necessary now.